File: kotlin/MarksAverage.kt
fun main() {
    println("Enter the first mark:")
    val mark1 = readLine()?.toIntOrNull() ?: 0

    println("Enter the second mark:")
    val mark2 = readLine()?.toIntOrNull() ?: 0

    println("Enter the third mark:")
    val mark3 = readLine()?.toIntOrNull() ?: 0

    val average = (mark1 + mark2 + mark3) / 3.0
    val roundedAverage = kotlin.math.round(average).toInt()

    if (roundedAverage >= 40) {
        println("PASS")
    } else {
        println("FAIL")
    }
}

File: swift/MarksAverage.swift
import Foundation

print("Enter the first mark:")
let mark1 = Int(readLine() ?? "") ?? 0

print("Enter the second mark:")
let mark2 = Int(readLine() ?? "") ?? 0

print("Enter the third mark:")
let mark3 = Int(readLine() ?? "") ?? 0

let average = Double(mark1 + mark2 + mark3) / 3.0
let roundedAverage = Int(average.rounded())

if roundedAverage >= 40 {
    print("PASS")
} else {
    print("FAIL")
}
